docs/sources/panels/queries.md

```diff
@@ -0,0 +1,120 @@
++++
+title = "Queries"
+type = "docs"
+[menu.docs]
+identifier = "queries"
+parent = "panels"
+weight = 300
++++
+
+# Queries
+
+> **Note:** This documentation refers to a feature only available in Grafana 7.0 beta.
+
+_Queries_ are how Grafana panels communicate with data sources to get data for the visualization. A query is basically a question written in the query language used by the data source. Grafana asks, "Hey data source, would you send me this data, organized this way?" If the query is properly formed, than the data source responds. How often the query is sent to the data source and how many data points are collected can be adjusted in the panel data source options.
+
+Grafana supports up to 26 queries per panel.
+
+## Query editors
+
+Query editors are forms that help you write queries. Depending on your data source, the query editor might completion, metric names, or variable suggestion.
```

Comment:
`might completion`? this feels weird

Reply:
Yeah, that sort of thing is what happens when I try to write documentation when my brain is fried. This is why we have other people edit. :)

Reply:
Fixed. Hopefully. Please make sure what I wrote makes sense.

Reply:
all good now!